templates/18_delete.go.tpl

```diff
@@ -211,6 +245,33 @@ func (o {{$alias.UpSingular}}Slice) DeleteAll({{if .NoContext}}exec boil.Executo
 	}
 	{{- end}}
 
+	{{if $soft -}}
```

Comment:
I missed the `slice.DeleteAll` in the previous PR and added it in, take note and review again pls, thanks!